README.md

```diff
@@ -93,12 +220,8 @@ This will compile the `arkd` and `ark` binaries for your current architecture. F
    ```sh
    cd server
    make build
-   cd ../client
```

Comment:
for the local dev setup maybe better to showcase `make run` and `make run-neutrino`.. sounds annoying to build the server to test a new dev feature